data/json/items/armor.json

```diff
@@ -10006,12 +10006,29 @@
         "flags" : ["VARSIZE"],
         "coverage" : 90,
         "material_thickness" : 1
+    },{
+        "type" : "ARMOR",
+        "id" : "tunic",
+        "name" : "tunic",
+        "weight" : 70,
```

Comment:
Not sure where I got the weight from. Overlooked it, it seems. What's the weight for sleeveless tunics and rag tunics?